Makefile

```
@@ -0,0 +1,32 @@
+include .docker.env
+
+run-devserver:
+  @echo "Updating the docker env variables..."
+  @echo "save_datastore        = $(save_datastore)"
+  @echo "disable_host_checking = $(disable_host_checking)"
+  @echo "no_auto_restart       = $(no_auto_restart)"
+  @echo "prod_env              = $(prod_env)"
+  @echo "maintenance_mode      = $(maintenance_mode)"
+  @echo "source_maps           = $(source_maps)"
```

```suggestion
  @echo "    save_datastore        = $(save_datastore)"
  @echo "    disable_host_checking = $(disable_host_checking)"
  @echo "    no_auto_restart       = $(no_auto_restart)"
  @echo "    prod_env              = $(prod_env)"
  @echo "    maintenance_mode      = $(maintenance_mode)"
  @echo "    source_maps           = $(source_maps)"
```

I like your change, but I was actually thinking of something like this, which makes it clear that the later lines are the variables mentioned in the first line

---

done